pyintegration/tests/test_table.py

```diff
@@ -394,7 +463,7 @@ def test_agg_all_by(self):
                 self.assertGreaterEqual(result_table.size, 1)
 
         with self.assertRaises(DHError) as cm:
-            test_table.agg_all_by(count_("aggCount"), ["grp_id"])
+            test_table.agg_all_by(count_("aggCount"), "grp_id")
```

Comment:
Should both of these be tested?

Reply:
yes, the single value case covers the original test and this change